src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter_Statics.cs

```diff
@@ -54,7 +54,7 @@ public static void CompileMethod(WebAssemblyCodegenCompilation compilation, WebA
                 // TODO: We should use the startup node to generate StartupCodeMain and avoid special casing here
```

Comment:
Can the TODO be removed?

Reply:
Not yet. This change moves us in that direction, but some of the library initializers crash before getting to main.